src/containers/PodcastPage/Podcast.tsx

```diff
@@ -0,0 +1,98 @@
+/* eslint-disable react/prop-types */
+import React from 'react';
+import {
+  AudioPlayer,
```

Comment:
Veldig mulig du må ha med initAudioPlayers her.

Reply:
Kan det være at dette kun er nødvendig for audioplayer som rendres i iframe?